packages/terra-dynamic-grid/src/Region.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  .region-child-container {
```

Comment:
can we add  a comment to why this is needed here

Reply:
Updated https://github.com/cerner/terra-core/pull/1043/commits/1d10ad3a4b1777c2555a6c0bee86be76e4f955f3#diff-765a8fc225f42601c30ea66ab554cb91R1